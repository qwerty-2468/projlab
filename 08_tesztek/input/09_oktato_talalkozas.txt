create room r1
create room r2
setroom r2 capacity 5

create student s1
create teacher t1
create student s2
create teacher t2
create cleaner c2

create rag rag
create rag r
create beerglass bg
create mask m
create falsemask fm
create sliderule sr
create falsesliderule fsr
create tvsz tv
create falsetvsz ftv
create airfresher a
create transistor t
create camembert c

neighbour r1 r2
addperson s1 r1
addperson t1 r1
addperson s2 r2
addperson t2 r2
addperson c2 r2

add rag s1
activate rag s1

add r r2
add bg r2
add m r2
add fm r2
add sr r2
add fsr r2
add tv r2
add ftv r2
add a r2
add t r2
add c r2

enter r2 s1
enter r2 t1

status r1
status r2
status t1
status t2
status