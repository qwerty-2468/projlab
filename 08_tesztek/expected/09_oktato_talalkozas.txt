
r1
neighbours: r2, 
peopleInRoom: 
itemsInRoom: 
gas: false
cursed: false
capacity: 3
curseActive: false
stickiness: 0

r2
neighbours: 
peopleInRoom: t2, c2, s1, t1, 
itemsInRoom: r, bg, m, fm, sr, fsr, tv, ftv, a, t, c, 
gas: false
cursed: false
capacity: 5
curseActive: false
stickiness: 2

t1
location: r2
itemsInHand: 
stunRemaining: 3

t2
location: r2
itemsInHand: 
stunRemaining: 3
Game in progress
